copy-mdn.js

```diff
@@ -0,0 +1,97 @@
+const bcd = require('@mdn/browser-compat-data')
+const { writeFileSync } = require('fs')
+
+/**
+ * This function maps the browser keys from @mdn/browser-compat-data, to caniuse's format.
+ */
+function bcdBrowserToCanIUseBrowser(bcdBrowser) {
+  let browser = bcdBrowser
+
+  if (browser === 'samsunginternet_android') {
+    browser = 'samsung'
+  } else if (browser === 'safari_ios') {
+    browser = 'ios_saf'
+  } else if (browser === 'opera_android') {
+    browser = 'op_mob'
+  } else if (browser === 'chrome_android') {
+    browser = 'and_chr'
+  } else if (browser === 'firefox_android') {
+    browser = 'and_ff'
+  } else if (browser === 'webview_android') {
+    browser = 'android'
+  }
+
+  return browser
+}
+
+/**
+ * This function maps support data from @mdn/browser-compat-data, to caniuse's
+ * format.
+ */
+function bcdDataToCanIUseData(bcdData, title) {
+  let result = {
+    title,
+    spec: bcdData.spec_url,
+    stats: {}
+  }
+
+  let supportData = bcdData.support
+
+  Object.keys(supportData).forEach(browser => {
+    let browserDataRaw = supportData[browser]
+    let browserData
+
+    // Browser support data in BCD can either be an object or an array.
+    if (Array.isArray(browserDataRaw)) {
+      /*
+       If the first entry in the array has a version added of "preview" we want to get the second entry.
+       This allows us to ignore browsers such as Safari Tech Preview.
+       */
+      if (browserDataRaw[0].version_added === 'preview') {
+        browserData = browserDataRaw[1]
+      } else {
+        browserData = browserDataRaw[0]
+      }
+    } else {
+      // If it's not an array it's already in the correct format to process.
+      browserData = browserDataRaw
+    }
+
+    result.stats[bcdBrowserToCanIUseBrowser(browser)] = {}
+    // Loop through all versions for the current browser
+    Object.keys(bcd.browsers[browser].releases).forEach(version => {
+      /**
+       * Feature is supported when:
+       * The BCD version isn't null or false (it exists)
+       * The BCD version isn't "preview", used for preview browsers such as Safari TP.
+       * The current version is greater than or equal to the BCD version.
+       * There's no prefix information in the BCD entry.
+       * There's no flag information in the BCD entry.
+       * The implementation isn't marked as partial.
+       */
+      let supported = browserData.version_added
+      supported &&= browserData.version_added !== 'preview'
+      supported &&= parseFloat(version) >= parseFloat(browserData.version_added)
+      supported &&= !browserData.prefix
```

Comment:
Do we need to add ` x` postfix to `result.stats[bcdBrowserToCanIUseBrowser(browser)][version]` when we need prefix?

Reply:
You can find these one-letter codes description here https://github.com/Fyrd/caniuse/blob/main/CONTRIBUTING.md

Reply:
Yeap, for Chrome 96 we need `"y x"` value.

Autoprefixer inserts prefix only if string contain `/ x($|\s)/`

Reply:
So I've changed the code to make it so if the support statement has a version_added entry and a prefix entry it will add the x otherwise it won't. 

So in the above example it should generate "n x" for all chrome versions up to and including Chrome 95, but for 96 onwards it will generate "y". Which I think is correct behaviour? If we need to handle situations where both prefix and unprefix is supported that will require changing the processing code?

Reply:
Can I Use uses `y x` if browser support a feature but need a prefix.

Reply:
Will double check the code, on second thoughts I think I may have been wrong above and it's behaving as required after my change

Reply:
`"a x"` is for partial support with prefix

Reply:
I've updated the code to handle these more complicated scenarios as well as some edge cases that can occur in MDN data such as "range" values (e.g. `≤18`) and `true` for version_added, and the version_removed column for if a feature is no longer supported.

Reply:
Looks good. Can you show the current output after decompressing?